css/selectors/focus-visible.json

```diff
@@ -37,7 +37,7 @@
               }
             ],
             "edge": {
```

Comment:
I think I poorly described the solution in the original issue. I think what needs to happen here is:

1. Copy the data from Chrome completely (i.e., the array for `"chrome"`, above). This way we'll have two support statements: one without a flag and one with.
2. For Edge only, change version "67" to "79", reflecting the first version Edge was based on Chromium.

Reply:
I think I understand. Just updated the PR. Please let me know if it is what is expected.